debian/patches/05-remove-config

```diff
@@ -0,0 +1,23 @@
+Index: sway-regolith/meson.build
+===================================================================
+--- sway-regolith.orig/meson.build
++++ sway-regolith/meson.build
+@@ -244,18 +244,6 @@ if get_option('swaynag')
+ 	subdir('swaynag')
+ endif
+ 
+-config = configuration_data()
+-config.set('datadir', join_paths(prefix, datadir))
+-config.set('prefix', prefix)
+-config.set('sysconfdir', join_paths(prefix, sysconfdir))
+-
+-configure_file(
+-	configuration: config,
+-	input: 'config.in',
+-	output: '@BASENAME@',
+-	install_dir: join_paths(sysconfdir, 'sway')
```

Comment:
Removing these configuration files can render the base session non-functional. Either we remove the desktop files to make the session inaccessible, or make sure the configs are available as well.